apps/Learning-Buddy/screens/LoginScreen.jsx

```diff
@@ -13,29 +13,41 @@ export const LoginScreen = ({ navigation }) => {
   const { signInWithEmail } = auth;
   const [email, setEmail] = useState();
   const [password, setPassword] = useState();
+  const [emailError, setEmailError] = useState(false);
+  const [passwordError, setPasswordError] = useState(false);
 
   const formVerify = () => {
-    if (email && password) {
-      if (isValidEmail(email)) {
-        const returnVal = {
-          email,
-          password
-        };
+    if (!email && !password) {
+      setEmailError(true);
+      setPasswordError(true);
+    }
```

Comment:
Same not as on the sign up screen, this one doesn't cause a visible issue since it's only two fields.

Reply:
This has been fixed